stable/redis/values-production.yaml

```diff
@@ -175,6 +175,13 @@ master:
     - ReadWriteOnce
     size: 8Gi
 
+  ## Update strategy, can be set to RollingUpdate or onDelete by default.
+  ## https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#updating-statefulsets
+  statefulset:
+    updateStrategy: onDelete
```

Comment:
It should be: `OnDelete`. Tests are failing because `onDelete` is not recognised as a valid value.